### ISSUE TITLE
Fluxgym is saying that training is complete far to quickly and produces no safetensor files.

### ISSUE THREAD

**Issue by 0010110000** [cocktailpeanut/fluxgym#465]
Whenever I try to train a lora with fluxgym, it almost instantly says that training is done but has obviously not trained anything and produced no lora files.

When I click “Train” get the following messages almost as soon as I click the button:

	info
	Training Complete. Check the outputs folder for the LoRA files.

	info
	Started training

	info
	Generated sample_prompts.txt

	info
	Generated dataset.toml

	info
	Generated train script at train.bat

I have an RTX 4060 Ti with 16gb vram and I have 32gb ram.

In the train log I get: “[ERROR] Command exited with code 1”

It is the only error I have noticed.

Does anyone know what could be causing this?

Train log:

	[2025-09-18 18:42:24] [INFO] Running W:\AI\FluxGym\fluxgym\outputs\honestabe\train.bat
	[2025-09-18 18:42:24] [INFO] 
	[2025-09-18 18:42:24] [INFO] (env) W:\AI\FluxGym\fluxgym>accelerate launch   --	mixed_precision bf16   --num_cpu_threads_per_process 1   sd-scripts/flux_train_network.py   --	pretrained_model_name_or_path "W:\AI\FluxGym\fluxgym\models\unet\flux1-dev.sft"   --	clip_l "W:\AI\FluxGym\fluxgym\models\clip\clip_l.safetensors"   --t5xxl "W:	\AI\FluxGym\fluxgym\models\clip\t5xxl_fp16.safetensors"   --ae "W:	\AI\FluxGym\fluxgym\models\vae\ae.sft"   --cache_latents_to_disk   --save_model_as 	safetensors   --sdpa --persistent_data_loader_workers   --max_data_loader_n_workers 2   --seed 	42   --gradient_checkpointing   --mixed_precision bf16   --save_precision bf16   --	network_module networks.lora_flux   --network_dim 4   --optimizer_type adafactor   --	optimizer_args "relative_step=False" "scale_parameter=False" "warmup_init=False"   --	split_mode   --network_args "train_blocks=single"   --lr_scheduler constant_with_warmup   --	max_grad_norm 0.0   --learning_rate 8e-4   --cache_text_encoder_outputs   --	cache_text_encoder_outputs_to_disk   --fp8_base   --highvram   --max_train_epochs 16   --	save_every_n_epochs 1   --dataset_config "W:	\AI\FluxGym\fluxgym\outputs\honestabe\dataset.toml"   --output_dir "W:	\AI\FluxGym\fluxgym\outputs\honestabe"   --output_name honestabe   --timestep_sampling 	shift   --discrete_flow_shift 3.1582   --model_prediction_type raw   --guidance_scale 1   --	loss_type l2
	[2025-09-18 18:42:24] [ERROR] Command exited with code 1
	[2025-09-18 18:42:24] [INFO] Runner: <LogsViewRunner nb_logs=4 exit_code=1>

Command console:

	current_account=None
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=A, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]	
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=Ab, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '',	 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=AbL, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 		False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=AbLn, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '',	 '', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=AbLncl, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=20G, sample_prompts=AbLncln, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=12G, sample_prompts=AbLncln, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=4, 	timestep_sampling=shift, guidance_scale=1, vram=12G, sample_prompts=AbLncln, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 	False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	gen_sh: network_dim:4, max_train_epochs=16, save_every_n_epochs=1, 	timestep_sampling=shift, guidance_scale=1, vram=12G, sample_prompts=AbLncln, 	sample_every_n_steps=0
	original_advanced_component_values = ['', False, False, False, '', '', False, '', False, False, '', '', '', 	'', '', '', '', '', '', False, '', '', '', '', False, '', False, '', '', False, False, '', False, False, False, False, '', 	False, False, '', False, False, False, False, False, False, '', '', '', '', '', '', '', '', '', '', '', '', '', False, '', '', 		False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', False, '', '', '', '', '', False, '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 	'', '', '', '', False, False, '', False, False, '', False, '', '', '', '', False, '', False, '', '', '', '', False, False, '', '', 	'', '', '', '', '', '', False, False, False, False, '', '', False, '', False, False, '', '', '', '', '', False, '', '', '', '', 	False, False, False, False, '', '', '', '', '', '', '', '', '', False, False, False, False, '', False]
	max_train_epochs=16 num_images=7, num_repeats=10, total_steps=1120
	Creating dataset
	resize datasets/honestabe\3e2dd303319da14153b26cc9f798ef89.jpg : 1024x1281
	image_path=datasets/honestabe\3e2dd303319da14153b26cc9f798ef89.jpg, caption_path = W:	\AI\FluxGym\fluxgym\datasets\honestabe\3e2dd303319da14153b26cc9f798ef89.txt, 	original_caption=AbLncln
	W:\AI\FluxGym\fluxgym\datasets\honestabe\3e2dd303319da14153b26cc9f798ef89.txt create 	a .txt caption file
	resize datasets/honestabe\16th-president-united-states-america-108062570.jpg : 1024x1410
	image_path=datasets/honestabe\16th-president-united-states-america-108062570.jpg, 	caption_path = W:\AI\FluxGym\fluxgym\datasets\honestabe\16th-president-united-states-	america-108062570.txt, original_caption=AbLncln
	W:\AI\FluxGym\fluxgym\datasets\honestabe\16th-president-united-states-	america-108062570.txt create a .txt caption file
	resize datasets/honestabe\200928_r37090.jpg : 1024x1414
	image_path=datasets/honestabe\200928_r37090.jpg, caption_path = W:	\AI\FluxGym\fluxgym\datasets\honestabe\200928_r37090.txt, original_caption=AbLncln
	W:\AI\FluxGym\fluxgym\datasets\honestabe\200928_r37090.txt create a .txt caption file
	resize datasets/honestabe\1698298016_myskillsconnect-com-p-linkoln-foto-v-tsilindre-32.jpg : 	1024x1460
	image_path=datasets/honestabe\1698298016_myskillsconnect-com-p-linkoln-foto-v-	tsilindre-32.jpg, caption_path = W:	\AI\FluxGym\fluxgym\datasets\honestabe\1698298016_myskillsconnect-com-p-linkoln-foto-v-	tsilindre-32.txt, original_caption=AbLncln
	W:\AI\FluxGym\fluxgym\datasets\honestabe\1698298016_myskillsconnect-com-p-linkoln-	foto-v-tsilindre-32.txt create a .txt caption file
	resize datasets/honestabe\Abraham_Lincoln_1863_Portrait_3x4_cropped.jpg : 1024x1365
	image_path=datasets/honestabe\Abraham_Lincoln_1863_Portrait_3x4_cropped.jpg, 	caption_path = W:	\AI\FluxGym\fluxgym\datasets\honestabe\Abraham_Lincoln_1863_Portrait_3x4_cropped.txt, 	original_caption=AbLncln
	W:	\AI\FluxGym\fluxgym\datasets\honestabe\Abraham_Lincoln_1863_Portrait_3x4_cropped.txt 	create a .txt caption file
	resize datasets/honestabe\DT1652-scaled-e1642296815665.jpg : 2055x1024
	image_path=datasets/honestabe\DT1652-scaled-e1642296815665.jpg, caption_path = W:	\AI\FluxGym\fluxgym\datasets\honestabe\DT1652-scaled-e1642296815665.txt, 	original_caption=AbLncln
	W:\AI\FluxGym\fluxgym\datasets\honestabe\DT1652-scaled-e1642296815665.txt create a .txt 	caption file
	resize datasets/honestabe\HD_LincolnA8Nov1863z.jpg : 1024x1527
	image_path=datasets/honestabe\HD_LincolnA8Nov1863z.jpg, caption_path = W:	\AI\FluxGym\fluxgym\datasets\honestabe\HD_LincolnA8Nov1863z.txt, 	original_caption=AbLncln
	W:\AI\FluxGym\fluxgym\datasets\honestabe\HD_LincolnA8Nov1863z.txt create a .txt caption 	file
	destination_folder datasets/honestabe
	concept_sentence=AbLncln
	lora_name HonestAbe, concept_sentence=AbLncln, output_name=honestabe
	license_items=['license: other', 'license_name: flux-1-dev-non-commercial-license', 	'license_link: https://huggingface.co/black-forest-labs/FLUX.1-dev/blob/main/LICENSE.md']
	license_str = license: other
	license_name: flux-1-dev-non-commercial-license
	license_link: https://huggingface.co/black-forest-labs/FLUX.1-dev/blob/main/LICENSE.md
	no samples

**Comment by Lethia-ai** [comment]
Same problem with RTX 5090, the graphics card doesn't work, only the processor, and it doesn't save any safetensor files.

Error: The 8-bit optimizer is not available on your device, only available on CUDA for now.
